networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -5,7 +5,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: CNI VRF plug-in
-include::modules/technology-preview.adoc[leveloffset=+1]
+[NOTE]
+====
+Using a VRF through the `ip vrf exec` command is not supported in {product-title} pods. To use VRF, applications can bind directly to the VRF interface.
```

Comment:
should this go in the module below `modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc` together with the other note?

Reply:
@fedepaol Sorry for the late Friday reply! I think you're right, I merged the two comments together in both sections. Could you take another look? Thank you!